.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
Sigh for https://gitlab.com/pycqa/flake8/-/issues/428

Reply:
Yep, but I did look before adding this.